app/components/Analyst/History/HistoryContent.tsx

```diff
@@ -383,47 +391,86 @@ const HistoryContent = ({ historyItem, prevHistoryItem }) => {
 
     return (
       <StyledContent data-testid="history-content-community-progress-report">
-        { op === 'INSERT' && prevHistoryItem?.record &&
-        <span>{displayName} updated a{' '}</span>
-        }
-        { op === 'INSERT' && prevHistoryItem?.record === undefined &&
-        <span>{displayName} created a{' '}</span>
-        }
-        { op === 'UPDATE' && record.history_operation === 'deleted' &&
-        <span>{displayName} deleted a{' '}</span>
-        }
+        {op === 'INSERT' && prevHistoryItem?.record && (
+          <span>{displayName} updated a </span>
+        )}
+        {op === 'INSERT' && prevHistoryItem?.record === undefined && (
+          <span>{displayName} created a </span>
+        )}
+        {op === 'UPDATE' && record.history_operation === 'deleted' && (
+          <span>{displayName} deleted a </span>
+        )}
         <b>Community Progress Report</b>
         <span> on {createdAtFormatted}</span>
-        
+
         {op === 'INSERT' && changedFile && (
           <HistoryFile
-          filesArray={record.json_data.progressReportFile || []}
-          title="Uploaded Community Progress Report Excel"
-        />
-        )} 
+            filesArray={record.json_data.progressReportFile || []}
+            title="Uploaded Community Progress Report Excel"
+          />
+        )}
         {op === 'INSERT' && showHistoryDetails && (
           <HistoryDetails
             json={record.json_data}
             prevJson={prevHistoryItem?.record?.json_data || {}}
-            excludedKeys={['ccbc_number','progressReportFile']}
+            excludedKeys={['ccbc_number', 'progressReportFile']}
             diffSchema={communityReportSchema}
-            overrideParent='communityReport'
+            overrideParent="communityReport"
+          />
+        )}
+      </StyledContent>
+    );
+  }
+
+  if (tableName === 'application_claims_data') {
+    const operation = historyItem.record?.history_operation;
+    const isUpdate = operation === 'updated';
+    const newFile = record.json_data?.claimsFile;
+    const oldFile = prevHistoryItem?.record?.json_data?.claimsFile;
+    const changedFile =
+      (isUpdate && oldFile && !newFile) ||
+      (newFile && !oldFile) ||
+      (newFile && oldFile && newFile[0].uuid !== oldFile[0].uuid);
+
+    return (
+      <StyledContent data-testid="history-content-claims">
+        <span>
+          {displayName} {operation} a <b>Claim & Progress Report</b> on{' '}
+          {createdAtFormatted}
+        </span>
+        {changedFile && (
+          <HistoryFile
+            filesArray={record.json_data.claimsFile || []}
+            title="Uploaded Claims & Progress Report Excel"
           />
-        )} 
+        )}
+        {showHistoryDetails && isUpdate && (
+          <HistoryDetails
+            json={record.json_data}
+            prevJson={prevHistoryItem?.record?.json_data || {}}
+            excludedKeys={['ccbc_number', 'progressReportFile']}
```

Comment:
This should be `excludedKeys={['ccbc_number', 'claimsFile']}` otherwise we get the file metadata being diffed.

Reply:
https://github.com/bcgov/CONN-CCBC-portal/pull/2402/commits/e944b6e4280524f5ccdfde7bc517f3ab31f7e1af

Fixed! Thanks!